libp2p/stream_muxer/mplex/mplex_stream.py

```diff
@@ -146,7 +146,7 @@ def is_initiator(self) -> bool:
         :return: number of bytes written
         """
         if self.event_local_closed.is_set():
-            raise MplexStreamClosed(f"cannot write to closed stream: data={data}")
+            raise MplexStreamClosed(f"cannot write to closed stream: data={data!r}")
```

Comment:
I'd recommend this one to be moved to another PR, it should be a very quick :+1:

Reply:
It seems like this is already resolved in `master` with d52b093286c9b4c33b26bb9d4af9ee5a24e52c71